packages/foodie-backend/src/middlewares/index.ts

```diff
@@ -0,0 +1,7 @@
+import attachCurrentUser from './attachCurrentUser';
```

Comment:
I dint think we should have na index entry point, if youre improrting a file, go to the file and import in directly

Reply:
I will remove this. Thought i removed it when i was doing the folder restructuring